sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/SimAdministrationConfiguration.kt

```diff
@@ -114,41 +114,72 @@ data class SwtHssConfig(
 ) : HssConfig(name = name)
 
 
-class ProfileVendorConfigException(msg: String) : Exception(msg)
-
 /**
  * Configuration for profile vendors.  The name is a name alphanumeric + undrescore)
  * the es2plus endpoint is an fqdn, with an optional portnumber.  Similarly for the es9plus
  * endpoint.  The requester identifier is a string that is intended to identify the requester,
  * obviously in addition to client certificate.
  */
-data class ProfileVendorConfig(
```

Comment:
Changed this from a data class to an ordinary class. I wanted to override the getter for es2plusEndpoint, but on second thoughts that seemed like a bad design, so I chose to make the es2plusEndpoint that is read from the config private, and only adcessible via the saeEndpoint, which removes everything after the hostname/port part of the URL.